### PR TITLE
 FLAOF405X8 config.h

### PR DESCRIPTION
**Pull request by vgpengke**
Add FLAO  F405X8V1 config.h

### CODE REVIEW

File: configs/FLAO/config.h [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+#define ESCSERIAL_PIN     PC11
+#define ADC_VBAT_PIN     PC2
+#define ADC_RSSI_PIN      PC3
+#define ADC_CURR_PIN    PC1
+#define PINIO1_PIN        PB9
+#define FLASH_CS_PIN      PC0
+#define MAX7456_SPI_CS_PIN PA13
+#define GYRO_1_EXTI_PIN    PC4
+#define GYRO_1_CS_PIN     PA4
+#define USB_DETECT_PIN   PB12
```
Comment (haslinghuis) [inline configs/FLAO/config.h:84]: Please fix Indentation

File: configs/FLAO/config.h [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+
+#define BOARD_NAME        F405X8V1
```
Comment (haslinghuis) [inline configs/FLAO/config.h:26]: Usually we don't use V1 but continue with V2 ...

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/FLAO/config.h (hunk 1 above)
Comment (vgpengke) [reply configs/FLAO/config.h]: fix Indentation

**Comment by haslinghuis** [comment]
1. Please apply for a `FLAO` manufacturer ID first.
2. Was suggesting to leave out V1 not renaming it to V2.
3. Then we need a better board name not starting with F405 as the name needs to be associated with the brand and reflected in the folder name where the config.h will reside.

**Comment by vgpengke** [comment]
> 1. 请先申请`FLAO`制造商ID。
> 2. 建议忽略 V1，不要将其重命名为 V2。
> 3. 然后，我们需要一个更好的主板名称，不能以 F405 开头，因为该名称需要与品牌相关联，并反映在 config.h 所在的文件夹名称中。

We are a new hardware producing factory in shenzhen,china.
We need apply the manufacture id for our  hardware board.
The name of the company selling the board and supporting customers is :FLAO Limited
The website URL for this company is :www.sunrisemodel.com
Board_name: FLAOF405X8
manufacturer_id:FLAO

File: configs/FLAO/config.h (hunk 2 above)
Comment (vgpengke) [reply configs/FLAO/config.h]: Have been modified

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**